code/modules/surgery/organs/stomach/external/_external_organs.dm

```diff
@@ -0,0 +1,219 @@
+/**
+* System for drawihg organs with overlays. Used as a replacement for the datumized species features, so that you can tear our a moths wings and stick it on a lizard
+* Works in tandem with the /datum/sprite_accessory datum to generate sprites
+* Unlike normal organs, we're actually inside a persons limbs at all times
+*/
+
+/obj/item/organ/external
+	name = "external organ"
+	desc = "An external organ that is too external."
+
+	///Sometimes we need multiple layers, for like the back, middle and front of the person
+	var/layers = list()
+
+	///Defines what kind of 'organ' we're looking at. Sprites have names like 'm_firemoth_mothwings'. 'mothwings' would then be preference
+	var/preference = ""
+	///Sprite datums we use to draw on the bodypart
+	var/list/sprite_datums = list()
+	///Key of the icon states of all the sprite_datums for easy caching
+	var/cache_key = ""
+
+	///Reference to the limb we're inside of
+	var/obj/item/bodypart/ownerlimb
```

Comment:
I don't see this reference being cleared. Are you sure this will not cause hard deletes?

Reply:
Good catch